libcloud/compute/drivers/equinixmetal.py

```diff
@@ -252,8 +252,12 @@ def ex_list_nodes_for_project(self, ex_project_id, include="plan", page=1, per_p
         return list(map(self._to_node, data))
 
     def list_locations(self):
+<<<<<<< HEAD
```

Comment:
Bad merge - several unresolved conflicts in this PR.

Reply:
Yes missed that file to be committed, resolved